lib/winston-syslog.js

```diff
@@ -342,9 +358,17 @@ class Syslog extends Transport {
     if (this.isDgram) {
       return this.connectDgram(callback);
     }
+    // Set Secure options for TLS
+    const options = {
+      servername: this.host,
+      checkServerIdentity: () => this.checkServerIdentity, // check FQDN
```

Comment:
I'm not seeing either of these in the options list at the documentation link provided. Am I missing something?

Reply:
I don't see documentation citing that the options for tls.connect can be used there.  Am I missing something that you might be able to point me to the specific language, if you still want these included?

Reply:
Yes, they don’t mention that. I have already used it in my project, which doesn’t raise errors. But for now, we can remove it. If it is needed, We will add it in the future.